reference/carts.v3.yml

```diff
@@ -512,7 +522,7 @@ paths:
                             message: Happy Birthday Jane!
                       created_time: '2018-09-17T14:27:39.000Z'
                       updated_time: '2018-09-17T14:53:40.000Z'
-                    meta: {}
+                      meta: {}
```

Comment:
`meta` should be at the same level of `data`

Reply:
Thank you